packages/amplify-provider-awscloudformation/src/aws-utils/aws-amplify.js

```diff
@@ -0,0 +1,55 @@
+const aws = require('aws-sdk');
+const proxyAgent = require('proxy-agent');
+const configurationManager = require('../../lib/configuration-manager');
+
+const amplifyServiceRegions = [
+  'us-east-1',
+  'us-east-2',
+  'us-west-2',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-central-1',
+  'ap-northeast-1',
+  'ap-northeast-2',
+  'ap-south-1',
+  'ap-southeast-1',
+  'ap-southeast-2',
+];
```

Comment:
could we have  single mapping file for all the services instead of it being scattered in different files.